sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6130,4 +6130,20 @@ ALTER TABLE person_history RENAME COLUMN namesofotherguardians TO namesofguardia
 
 INSERT INTO schema_version (version_number, comment) VALUES (296, 'Change namesOfOtherGuardians to namesOfGuardians #3413');
 
+-- 2020-12-03 Remove hospital from event's type of place #3617
+UPDATE location
```

Comment:
The SQL script doesn't run through (very likely the same for the app)

Reply:
My bad, this syntax is valid on SQLServer, but not on PGSQL.